statsmodels/discrete/tests/test_conditional.py

```diff
@@ -293,12 +292,12 @@ def test_conditional_mlogit1():
                     [0.65190315, 0.59653771]]),
         rtol=1e-5, atol=1e-5)
 
-def test_conditional_mlogit2():
+def test_conditional_mnlogit2():
 
-    df = gen_mlogit(90)
+    df = gen_mnlogit(90)
     df["x3"] = np.random.normal(size=df.shape[0])
-    model1 = ConditionalMlogit.from_formula("y ~ 0 + x1 + x2 + x3",
-              time="time", groups="g", data=df)
+    model1 = ConditionalMNLogit.from_formula("y ~ 0 + x1 + x2 + x3",
+              groups="g", data=df)
     result1 = model1.fit()
 
     # Regression tests
```

Comment:
this is my first time looking at this file, so just noticed the result1.summary() smoketest below.  can that be separated out and marked with a pytest.mark.smoke?  (OK to save for a subsequent PR, which I am willing to do)